library/core/src/main/java/com/google/android/exoplayer2/video/DummySurface.java

```diff
@@ -152,9 +153,16 @@ private static void assertApiLevel17OrHigher() {
    *
    * @param context Any {@link Context}.
    */
-  @SuppressWarnings("unused") // Context may be needed in the future for better targeting.
+  @SuppressWarnings("unused")
   private static boolean deviceNeedsSecureDummySurfaceWorkaround(Context context) {
-    return Util.SDK_INT == 24 && "samsung".equals(Util.MANUFACTURER);
+    return (Util.SDK_INT == 24 && "samsung".equals(Util.MANUFACTURER))
+            || (Util.SDK_INT >= 24 && Util.SDK_INT < 26
```

Comment:
I reverted this change by accident (sorry!), although having looked at it some more I'm not sure it's right that this method returns false on API levels below 24 just because we can't check the VR flag. If secure dummy surface didn't work reliably on devices without the VR flag on API levels 24 and 25, I doubt it worked reliable prior to the flag even having existed :).

It's likely that `EGL_EXT_protected_content` wasn't advertised prior to API level 24, so perhaps this is a hypothetical case, but seems worth disabling anyway. I'll push a new CL to un-revert and make this change (and will ref it in the related issue). Thanks!

Reply:
No probs, whatever works is good by me. Previously it would only ever return `true` if the SDK was 24 and it was a Samsung device, so I figured it was safe to return false <24 until we found a <24 device that required the secure dummy surface workaround.